testing/benchmark/system-profiles/16GBx2zone.tfvars

```diff
@@ -10,5 +16,8 @@ elasticsearch_zone_count = 2
 apm_server_size = "16g"
 # Number of shards for the ES indices
 apm_shards = 4
-# Benchmarks executor size executor
-worker_instance_type = "c6i.2xlarge"
+
+# Standalone
+
+standalone_apm_server_instance_size = "c6i.4xlarge"
+standalone_moxy_instance_size       = "c6i.8xlarge"
```

Comment:
Wow these are pretty big and beefy. Maybe we could add a comment here. Also looking at the size, `c6i.4xlarge` seems to be a 32GB instance, not 16GB. Is that what you intended?

Reply:
good catch, the sizing is fixed now